src/internal/transport/AbstractRequestProcessor.ts

```diff
@@ -299,8 +301,8 @@ export abstract class AbstractRequestProcessor implements RequestProcessor {
                              ts: number = Date.now()) {
         return {
             correlationId: _.get(request, "correlation_id") || _.get(request, "extensions.correlation_id"),
-            teamId: _.get(request, "team.id") || _.get(request, "extensions.team_id"),
-            teamName: _.get(request, "team.name") || _.get(request, "extensions.team_name"),
+            workspaceId: _.get(request, "team.id") || _.get(request, "extensions.team_id"),
+            workspaceName: _.get(request, "team.name") || _.get(request, "extensions.team_name"),
```

Comment:
Has the request object not been updated to use workspace?

Reply:
No